### DISCUSSION TITLE
[Help] Getting zk to work on Windows

### DISCUSSION THREAD

**Discussion by dc6c** [zk-org/zk#437] [Help]
### Verify

- [X] I searched the existing discussions for help

### How can we help you?

The maintainers have stated that `zk` is not supported on Windows, however in the Issues I have found guidance on building it from source. That works, and I can create and browse notes (running in Git Bash), but I can't figure out how to get the editor integration working. I have vim and VSCode available, and have tried using them as the value for the "editor" entry, both just names and with full paths, in Git Bash, Powershell, and Cmd shells, but I always get an error ""zk: error: failed to launch editor: ...".

A [question](https://github.com/zk-org/zk/issues/320) about editor integration on windows in the repo issues, but maybe this is a place where someone in the community can help with the advice?

**Comment by tjex** [comment]
Yeah, it's a bummer we don't have the resources to support windows 😞

Perhaps we can get a little further though. 
With what / how are you creating and browsing notes? You must be using an editor for that?
I'm just confused that you're getting the "failed to launch editor", but are able to create and browse (and edit?) notes.
